lib/ui/pages/location_page.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:food_app/utils/app_color.dart';
+import 'package:food_app/utils/app_typography.dart';
```

Comment:
We can change these 3 lines to just one line with `part of 'pages.dart'` and also added `part of 'location_page.dart'` in `ui/pages/pages.dart`